internal/fsutil/create_test.go

```diff
@@ -44,6 +47,7 @@ var createTests = []createTest{{
 		"/foo/bar": "symlink ../baz",
 	},
 }, {
+	summary: "Create a subdirectory",
```

Comment:
```suggestion
	summary: "Create a directory",
```
Technically you are creating two directories but this is concise enough.

Reply:
Fixed in [3e8d872](https://github.com/letFunny/chisel/pull/17/commits/3e8d872c21ef3ed81e75c5319a052f872feb8b25).